src/server/views/dashboard/partials/list-item-tag.njk

```diff
@@ -1,21 +1,13 @@
 {% set colours = {
-  published: 'green',
-  approved: 'turquoise',
-  'to_do': 'blue',
-  'out_with_provider': 'yellow'
+  live: 'green',
+  'new': 'grey',
+  'unpublished': 'red',
+  'archived': 'grey'
   }
```

Comment:
Just a heads up, [in this PR](https://github.com/UKForeignOffice/lists/pull/490/files#diff-12b5be4cb737b6662b9f5f7eae42666022b3e0ae655f179dda04c6034d9708f4), I've moved this object from the template to [the controller](https://github.com/UKForeignOffice/lists/pull/490/files#diff-916bf0afcf17bd341b29d6387320b07270771e5e3a69688be824fd1809093d56R49). It might cause some conflicts if that gets merged in before this.